sdk/schemaregistry/schema-registry-avro/package.json

```diff
@@ -89,8 +90,10 @@
     "@rollup/plugin-replace": "^2.2.0",
     "@types/chai": "^4.1.6",
     "@types/chai-as-promised": "^7.1.0",
+    "@types/lru-cache": "~5.1.1",
```

Comment:
Any reason to use the ~ minor selector here?

Reply:
nope, was the rush add default. I updated it.